Formula/thrift.rb

```diff
@@ -36,9 +36,11 @@ def install
       --with-openssl=#{Formula["openssl"].opt_prefix}
       --without-erlang
       --without-haskell
+      --without-java
```

Comment:
Why are you removing Java, I think it is important to have the Java bindings. If so, you need to remove the env-settings as well.

Reply:
Barely anyone uses it based on the analytics above and it pulls in a huge dependency, I'd say this is a valid reason to remove it.